apps/nsc-vfio/nsc.yaml

```diff
@@ -21,6 +21,10 @@ spec:
           image: artgl/dpdk-pingpong:latest
           imagePullPolicy: IfNotPresent
           command: ["/bin/bash", "-c", "sleep 60m"]
+          securityContext:
+            capabilities:
+              add:
+                - SYS_RESOURCE
```

Comment:
Why do we need CAP_SYS_RESOURCE?  Why didn't we before?

Reply:
Fixed - https://github.com/networkservicemesh/deployments-k8s/pull/7220